.github/workflows/python.yaml

```diff
@@ -31,7 +31,14 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version: ["3.10", "3.11", "3.12"]
-    name: Run tests on Python ${{ matrix.python-version }} on ${{ matrix.os }}
+        deps:
+          # Oldest supported versions
+          - lakefs: "lakefs==0.2.0"
+            fsspec: "fsspec==2023.12.0"
+          # Latest versions
+          - lakefs: "lakefs>0"
+            fsspec: "fsspec>0"
+    name: Run tests on Python ${{ matrix.python-version }} on ${{ matrix.os }} with lakefs ${{ matrix.deps.lakefs }} and fsspec ${{ matrix.deps.fsspec }}
```

Comment:
This dupes fsspec and lakefs, since they appear in the deps entries as well.

Reply:
Fixed in [e042709](https://github.com/aai-institute/lakefs-spec/pull/293/commits/e042709def22771a9f444124c6f934b3f554a3cf)